tests/fixtures/motor/generic_motor_fixtures.py

```diff
@@ -28,3 +28,27 @@ def generic_motor():
     )
 
     return motor
+
+
+@pytest.fixture
+def generic_motor_cesaroni_M1520():
+    """Defines a Cesaroni M1520 motor for the Prometheus rocket using the
+    GenericMotor class.
+
+    Returns
+    -------
+    GenericMotor
+        The Cesaroni M1520 motor for the Prometheus rocket.
+    """
+    return GenericMotor(
+        # burn specs: https://www.thrustcurve.org/simfiles/5f4294d20002e900000006b1/
```

Comment:
It is a good practice for us to document our acceptance tests source, specially for future refactors or corrections that require reading these data again.

Do you think it is helpful putting the source data for the tests in their docstring (possibly in a section of their own)?

Reply:
What do you mean by "source data for the tests" ?